demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c

```diff
@@ -110,7 +110,7 @@ typedef struct ProvisionedState_t
                           * in the subject of the device certificate. */
 } ProvisionedState_t;
 
-/* This function can be found in libraries/3rdparty/mbedtls/utils/mbedtls_utils.c. */
+/* This function can be found in libraries/3rdparty/mbedtls_utils/mbedtls_utils.c. */
```

Comment:
Correct, this is not part of the submodule. I have it in the 3rd party directory because the function that is being referenced is a copy of preexisting code in the mbed tls submodule. It may need to be updated as we upgrade to different versions of mbed tls. I don't use the original code because the file containing that function also has a main function. 

I'm using the path to the mbedtls_utils.c file because that is the file that will actually be linked.